pkg/loop/internal/pb/chain_reader.proto

```diff
@@ -95,7 +100,7 @@ message Sequence {
 // BoundContract represents a [github.com/smartcontractkit/chainlink-common/pkg/types.BoundContract].
 message BoundContract {
   string address = 1;
-  string name = 2;
+  string contract = 2;
```

Comment:
Why? Maybe at least contract_name?

Reply:
It was left this way because I added `method`, but then removed it.